src/libs/antares/study/area/list.cpp

```diff
@@ -1079,12 +1079,23 @@ static bool AreaListLoadFromFolderSingleArea(Study& study,
             area.thermal.list.enableMustrunForEveryone();
     }
 
+    // Short term storage
+    {
+        buffer.clear() << study.folderInput << SEP << "st-storage" << SEP << "series"
+            << SEP << area.id;
+        logs.debug() << "Loading series for st storage " << buffer;
+        ret = area.shortTermStorage.loadSeriesFromFolder(buffer.c_str()) && ret;
+        ret = area.shortTermStorage.validate() && ret;
+    }
+
     // Renewable cluster list
     if (study.header.version >= 810)
     {
         buffer.clear() << study.folderInput << SEP << "renewables" << SEP << "series";
         ret = area.renewable.list.loadDataSeriesFromFolder(study, options, buffer) && ret;
     }
+    if (!ret)
```

Comment:
```C++
if (!ret)
   logs.error() << "Error while loading sts series";
```
Why not right after the ST storage load ? **ret == false** here could mean there were a problem when loading renewable clusters, not when loading ST storage time series.

Reply:
Changed in #1239